source/_docs/unsupported-modules-plugins.md

```diff
@@ -220,9 +222,14 @@ $conf[‘schema_suppress_type_warnings’] = TRUE;
 ```
 
 <hr>
-### [Taxonomy CSV](https://www.drupal.org/project/taxonomy_csv)  
+### [Taxonomy CSV](https://www.drupal.org/project/taxonomy_csv)
 **Issue**:  This module requires the use of the `/tmp` directory. See [Using the tmp Directory](/docs/unsupported-modules-plugins/#using-the-tmp-directory) section below.
 <hr>
+
+### [Twig Extensions](https://www.drupal.org/project/twig_extensions)  
+**Issue**:  This module uses `php-intl`, which is not currently supported by Pantheon.
```

Comment:
The link is:  http://php.net/manual/en/intro.intl.php

@rachelwhitton  I asked Omar if he could have them test.  I would suggest we push this out as listed, that its not supported, and then we come back and modify it if the workaround is successful. Otherwise, this PR may be sitting for a while.